mnemosyne/diagnose.py

```
@@ -249,7 +252,10 @@ def log(category: str, check: str, status: str, detail: str = ""):
     # --- Database state ---
     try:
         from mnemosyne.core.memory import Mnemosyne
-        mem = Mnemosyne()
+        if bank:
+            mem = Mnemosyne(session_id="hermes_default", bank=bank)
+        else:
+            mem = Mnemosyne()
```

Ruff (0.15.20)

[warning] Use ternary operator `mem = Mnemosyne(session_id="hermes_default", bank=bank) if bank else Mnemosyne()` instead of `if`-`else`-block

Replace `if`-`else`-block with `mem = Mnemosyne(session_id="hermes_default", bank=bank) if bank else Mnemosyne()`

(SIM108)